File: BEmu/BloombergTypes/SessionOptions.cs
//------------------------------------------------------------------------------
// <copyright project="BEmu" file="BloombergTypes/SessionOptions.cs" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

namespace BEmu
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Text;

    public class SessionOptions
    {
        public string ServerHost { get; set; }
        public int ServerPort { get; set; }

        public SessionOptions() { }
    }
}

File: BEmuJava/bemu/src/main/java/com/bloomberglp/blpapi/SessionOptions.java
//------------------------------------------------------------------------------
// <copyright project="BEmu_maven" file="/BEmu_maven/bemu/src/main/java/com/bloomberglp/blpapi/SessionOptions.java" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

package com.bloomberglp.blpapi;

public class SessionOptions
{
	@SuppressWarnings("unused")
	private String _serverhost;
	public void setServerHost(String host)
	{
		this._serverhost = host;
	}
	
	@SuppressWarnings("unused")
	private int _serverPort;
	public void setServerPort(int port)
	{
		this._serverPort = port;
	}
}
